multicluster/controllers/multicluster/commonarea/remote_common_area.go

```diff
@@ -279,27 +277,15 @@ func (r *remoteCommonArea) updateRemoteCommonAreaStatus(connected bool, err erro
 	}
 }
 
-func (r *remoteCommonArea) updateLeaderStatus(leaderID common.ClusterID) {
+func (r *remoteCommonArea) updateLeaderStatus() {
 	defer r.mutex.Unlock()
 	r.mutex.Lock()
 
-	if leaderID == common.InvalidClusterID {
```

Comment:
The `InvalidClusterID` will be set only when the ClusterSet is not created and common area is nil or the ClusterSet is deleted and common area is reset to nil. So there is no case to call `updateLeaderStatus` when the leader ID is `InvalidClusterID`.